examples/dhcpd.conf

```diff
@@ -0,0 +1,79 @@
+##
+##  ISC-DHCPD configuration
+##
+
+# General Config
+
+default-lease-time 86400;
+max-lease-time 86400;
+
+
+# Define Mitel Specific Options
+
+option space sipdect;
+
+option sipdect.omm-ip code 10 = ip-address;
+option sipdect.omm-ip2 code 19 = ip-address;
+option sipdect.omm-syslog-ip code 14 = ip-address;
+option sipdect.omm-syslog-port code 15 = unsigned integer 16;
+option magic_str code 224 = text;
+option tftp-list code 150 = array of ip-address;
+option vlanid code 132 = unsigned integer 16;
```

Comment:
This might deserve a comment…
Maybe # Make RFPs switch to a tagged VLAN; the VLAN ID is stored in the RFP and used directly for all future DHCP requests

Reply:
You're right, some of the vendor specific options definitly need some comment.